sdk/src/lib/imagekit-angular.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Inject, Injectable } from '@angular/core';
+import { ImageKitConfiguration } from './utility/ik-type-def-collection';
+import { IMAGEKIT_CONFIG } from './imagekit-angular.tokens';
+const SDK_VERSION = '6.0.0';
+
+
+@Injectable()
+export class ImagekitService {
+
+  private configuration: ImageKitConfiguration;
+  
+  constructor(
+    @Inject(IMAGEKIT_CONFIG) config: ImageKitConfiguration
+  ) {
+    this.configuration = {
+      ...config,
+      sdkVersion: `angular-${SDK_VERSION}`
+    };
+  }
+
+  get ikInstance(): { options: ImageKitConfiguration } {
+    return {
+      options: this.configuration,
+    };
+  }
+
+  get _ikInstance(): { options: ImageKitConfiguration } {
```

Comment:
Why is this needed separately when it is just calling `ikInstance`?
Let's just keep one getter for this service.